### ISSUE TITLE
Scripts/markdown-enum has wrong hashbang in PyPI

### ISSUE THREAD

**Issue by mike-rsi** [A4Vision/enumerate-markdown#2]
The file `markdown-enum` that gets installed to the Python `Scripts` directory has a wrong hashbang which causes a `bad interpreter` error:

`bash: ./markdown-enum: /home/bugabuga/anaconda3/bin/python: bad interpreter: No such file or directory`

Top of `markdown-enum` when `pip install`'d:
```
#!/home/bugabuga/anaconda3/bin/python
import sys
import enumerate_markdown
...
```

Note: this doesn't match what's actually in the source repository: https://github.com/A4Vision/enumerate-markdown/blob/master/bin/markdown-enum#L1

Either the wrong file got manually uploaded to PyPI, or there's an error in the build process, or ...?

Manually editing the file after installation and replacing line #1 with `#!/usr/bin/env python` fixes the problem.

Note: this is on Windows 10 running under Git bash, pip 19.1.1 and Python 3.7.

**Comment by NorHei** [comment]
Ping....

Same with Ubuntu.

**Comment by A4Vision** [comment]
@NorHei @mike-rsi  thx
resolved in new version 1.0.0
